src/main/java/frc/robot/EagleUtil.java

```diff
@@ -196,10 +202,55 @@ public static ArrayList<Pose2d> calculateRedReefSetPoints() {
     DogLog.log("Caculation/Red Set Points", redPoses);
 
     m_redPoses = new ArrayList<Pose2d>(Arrays.asList(redPoses));
-
     return new ArrayList<Pose2d>(m_redPoses);
   }
 
+  public static ArrayList<Pose2d> calculateBlueAlgaeSetPoints() {
+    if (m_blueAlgaePoses != null) {
+      return new ArrayList<>(m_blueAlgaePoses);
+    }
+
+    Rotation2d sixty = Rotation2d.fromDegrees(60);
+    Pose2d startPose =
+        new Pose2d(X, Y - REEF_TO_REEF_DISTANCE / 2, Rotation2d.kZero); // Centered on first side
+
+    for (int i = 0; i < blueAlgaePoses.length; i++) {
+      blueAlgaePoses[i] = startPose.rotateBy(sixty.times(i));
+    }
+
+    for (int i = 0; i < blueAlgaePoses.length; i++) {
+      blueAlgaePoses[i] = blueAlgaePoses[i].relativeTo(BLUE_REEF_INVERT);
+    }
+
+    DogLog.log("Calculation/Blue Algae", BLUE_REEF);
+    DogLog.log("Calculation/Blue Algae Setpoints", blueAlgaePoses);
+    m_blueAlgaePoses = new ArrayList<>(Arrays.asList(blueAlgaePoses));
+    return new ArrayList<>(m_blueAlgaePoses);
+  }
+
+  public static ArrayList<Pose2d> calculateRedAlgaeSetPoints() {
+    if (m_redAlgaePoses != null) {
+      return new ArrayList<>(m_redAlgaePoses);
+    }
+
+    Rotation2d sixty = Rotation2d.fromDegrees(60);
+    Pose2d startPose =
+        new Pose2d(X, Y - REEF_TO_REEF_DISTANCE / 2, Rotation2d.kZero); // Centered on first side
+
+    for (int i = 0; i < redAlgaePoses.length; i++) {
+      redAlgaePoses[i] = startPose.rotateBy(sixty.times(i));
+    }
+
+    for (int i = 0; i < redAlgaePoses.length; i++) {
+      redAlgaePoses[i] = redAlgaePoses[i].relativeTo(RED_REEF_INVERT);
+    }
```

Comment:
We could probably merge this loop with the one above to save time on calculation (unless it specifically has to be done on a separate loop)